src/setting.rs

```diff
@@ -333,23 +333,7 @@ mod tests {
     fn test_find_setting() {
         assert_eq!(
             Setting::find("solution", "soln_freq"),
-            Some(&Setting {
-                name: "soln_freq".into(),
-                group: "solution".into(),
-                kind: SettingKind::Integer,
-                readonly: false,
-                expert: false,
-                units: Some("Hz".into()),
-                default_value: Some("10".into()),
-                description: Some("The frequency at which GNSS navigation solution is computed.\n".into()),
-                notes: Some("Minimum is 1 Hz. Maximum is 10 Hz for RTK positioning with a maximum of 15 used \
-                satellites. \nAt 5 Hz and lower the maximum number of used satellites is 22. 20 Hz is an absolute \
-                maximum with \na limit of 5 used satellites.\n\nSystem with inertial fusion (Duro Inertial, Piksi \
-                Multi Inertial) can output position at a higher rate \nthan the GNSS-only solution. See \
-                fused_soln_freq in the INS group.\n".into()),
-                enumerated_possible_values: None,
-                digits: None,
-            })
+            Some(&Setting { name: "soln_freq".to_string(), group: "solution".to_string(), kind: SettingKind::Integer, expert: false, readonly: false, description: Some("The frequency at which GNSS navigation solution is computed.\n".to_string()), default_value: Some("10".to_string()), notes: Some("Minimum is 1 Hz. Maximum is 10 Hz for RTK positioning with a maximum of 15 used satellites.\nAt 5 Hz and lower the maximum number of used satellites is 22. 20 Hz is an absolute maximum with\na limit of 5 used satellites.\n\nSystem with inertial fusion (Duro Inertial, Piksi Multi Inertial) can output position at a higher rate\nthan the GNSS-only solution. See fused_soln_freq in the INS group.\n".to_string()), units: Some("Hz".to_string()), enumerated_possible_values: None, digits: None })
```

Comment:
does this need clippy / fmt ?

Reply:
Apparently not, I ran `cargo fmt` and it didn't touch it.